_data/locales/tw.yml

```diff
@@ -0,0 +1,19 @@
+script_direction: ltr
+project:
+  title: Elixir School
+  description: Elixir 程式語言課程
+table_of_contents: 目錄
+share: 分享本頁面
+sections:
+  home: Home
+  basics: 基礎
+  advanced: 進階
+  specifics: 特色
+  libraries: 函式庫
+version_messages:
+  outdated: "Some contents of this translation may be outdated."
```

Comment:
@lithiumpie did you want to translate these phrases?

Reply:
Thanks for informing me of it. I will submit the translation later. 👍